rescript/orient.py

```diff
@@ -19,50 +18,34 @@ def orient_seqs(sequences: DNAFASTAFormat,
                 threads: int = 1,
                 left_justify: bool = False,
                 ) -> (DNAFASTAFormat, DNAFASTAFormat):
-    matched_temp, notmatched = DNAFASTAFormat(), DNAFASTAFormat()
+    oriented, notmatched = DNAFASTAFormat(), DNAFASTAFormat()
```

Comment:
Just wanted to point out that `vsearch orient` can handle both fastq and fasta files. This can likely wait for a separate PR. But we'd have to figure out a way to handle paired-end reads. That is, if we orient the forward read, then the reverse read should be oriented too.